fw-child/acf-json/group_674d41008a9cb.json

```diff
@@ -0,0 +1,75 @@
+{
+    "key": "group_674d41008a9cb",
+    "title": "Variable Meta",
+    "fields": [
+        {
+            "key": "field_674d4100b944a",
+            "label": "Variable ID",
+            "name": "var_name",
+            "aria-label": "",
+            "type": "text",
+            "instructions": "",
```

Comment:
Instructions are always recommended, such as stating to use a unique ID without any spaces.